### PR TITLE
Feature v0.14.5 cert ha support

### PR DESCRIPTION
**Pull request by afeiszli**

### CODE REVIEW

**Comment by afeiszli** [comment]
@mattkasun please review, this logic will add any key generated to the DB, and servers will only read their keys from the DB.

File: serverctl/tls.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
+package serverctl
```
Comment (mattkasun) [inline serverctl/tls.go:1]: why in package serverctl rather than adding to package tls

File: tls/tls.go [hunk 2]
```diff
@@ -17,8 +17,23 @@ import (
 	"golang.zx2c4.com/wireguard/wgctrl/wgtypes"
 )
 
-// CERTTIFICAT_VALIDITY duration of certificate validity in days
-const CERTIFICATE_VALIDITY = 365
+const (
```
Comment (mattkasun) [inline tls/tls.go:20]: some unused func in this package

File: serverctl/tls.go (hunk 1 above)
Comment (afeiszli) [reply serverctl/tls.go:1]: The new functions Use the database package, and TLS package gets imported by netclient. This avoids adding the database package as a dependency to netclient

File: tls/tls.go (hunk 2 above)
Comment (afeiszli) [reply tls/tls.go:20]: We saw the unused funcs but decided to leave them in in case you saw a need for them, but can take them out

**Pull request merged**